server/start.go

```diff
@@ -165,7 +165,7 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 	cmd.Flags().Uint64(FlagStateSyncSnapshotInterval, 0, "State sync snapshot interval")
 	cmd.Flags().Uint32(FlagStateSyncSnapshotKeepRecent, 2, "State sync snapshot to keep")
 
-	cmd.Flags().Bool(FlagIAVLFastNode, true, "Enable fast node for IAVL tree")
+	cmd.Flags().Bool(FlagIAVLFastNode, true, "Disable fast node for IAVL tree, set to 'false' to enable")
```

Comment:
```suggestion
	cmd.Flags().Bool(FlagIAVLDisableFastNode, false, "Disable fast node for IAVL tree, set to 'false' to enable")
```

Reply:
Should probably change the default value of `IAVLDisableFastNode` in `DefaultConfig` as well.

Reply:
IAVL fast tree is disabled by default, so renaming to `FlagIAVLDisableFastNode` would do?

Reply:
will need core team members to chime in, but I think it's supposed to be enabled by default, which is the case with main branch and 0.46.

Reply:
The default value should remains `true` since current (v0.45.10) default behavior is `disable Fast Node`.

https://github.com/cosmos/cosmos-sdk/blob/v0.45.10/simapp/simd/cmd/root.go#L279
```Go
baseapp.SetIAVLDisableFastNode(cast.ToBool(appOpts.Get(server.FlagIAVLFastNode)))
```

https://github.com/cosmos/cosmos-sdk/blob/v0.45.10/store/rootmulti/store.go#L41
```Go
const iavlDisablefastNodeDefault = true
```

Reply:
I think the bool flag can only be turn on in cli by passing `--iavl-disable-fastnode`, can one pass it like `--iavl-disable-fastnode=true`?

Reply:
`--iavl-disable-fastnode` should a shorthand for `--iavl-disable-fastnode=true`, so `--iavl-disable-fastnode=false` is possible

Reply:
BTW this CLI flag seems missing in 0.46, but the config is still there

Reply:
But the default behavior is different for >= 0.46
0.46 enables Fast Node by default (i.e. `disableFastNode == false`)

Reply:
I don't see the issue:

## 0.45
```go
cmd.Flags().Bool(FlagDisableIAVLFastNode, true, "Disable IAVL fast node")
```

### 0.46+
```go
cmd.Flags().Bool(FlagDisableIAVLFastNode, false, "Disable IAVL fast node")
```

Not having the flag in version after 0.45 is a CLI breaking change, so I think it is better to still keep the flag (true and false keep the same meaning anyway). We don't need to precise the default value in the message, as cobra does that already.

Reply:
@julienrbrt https://github.com/cosmos/cosmos-sdk/pull/13656 Please check if I understand correctly, then we can close this PR and follow up in the new one